pandas-stubs/plotting/_core.pyi

```diff
@@ -83,6 +84,54 @@ def boxplot(
     return_type: Literal["both"],
     **kwargs,
 ) -> _BoxPlotT: ...
+@overload
+def boxplot_frame_groupby(
```

Comment:
This function is public. It is re-exported (included in `__all__`) in the `pandas.plotting` module which is documented as _"Plotting public API"_. The function is also used as alias for a public method on the dataframe groupby class.

Reply:
Our philosophy is to not publish things that are not documented.  While it is used as an alias for the public method, the `groupby` stub has `boxplot` defined, so we should keep it defined there.

Reply:
Thank you for the second pass of review; I'll address your comments soon.

-------------

While I understand the philosophy of not including everything upstream in the stubs, I am having a hard time understanding the rational behind this specific case. The function is explicitly public, included in `__all__` of a module documented as "Plotting public API", mentioned explicitly in this module's docstring:
> backends are expected to implement the next functions:
> ...
> - boxplot_frame and boxplot_frame_groupby

and doesn't require any added maintenance in the stub since it has to be defined anyway for its use as a method of dataframe groupby. Could you please explain why you see this as a negative.

Reply:
It probably is in `__all__` so that other code from within pandas can import it but it isn't documented publicly https://pandas.pydata.org/docs/search.html?q=boxplot_frame_groupby I think pandas symbols are considered public only if they are publicly documented

Reply:
> I think pandas symbols are considered public only if they are publicly documented

This is the rule we've used with pandas-stubs.  We don't want users to call `boxplot_frame_groupby()`, so a user with a call to that function should see it as an error, as it would be unsupported.